.gitignore

```diff
@@ -8,6 +8,7 @@ result
 *.pending-snap
 *.snap*
 !cli/tests/cli-reference@.md.snap
+.cargo/insta.yaml
```

Comment:
You mean `.config/insta.yaml`?
https://insta.rs/docs/settings/#tool-config-file

Reply:
Whoops! Fixed. Thanks